draft-ietf-jsonpath-base.md

```diff
@@ -1446,6 +1447,7 @@ A Normalized Path is a JSONPath with restricted syntax that identifies a node by
 the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
 `$['book'][3]` and `$['book'][5]`. For a given JSON value, there is a one to one correspondence between the value's
 nodes and the Normalized Paths that identify these nodes.
+Note that there is at precisely one Normalized Path that identifies each node.
```

Comment:
s/at //

Reply:
Sorry, don't understand the problem with `"$" + segments.map { "['" + _1 + "']" }.join`

Reply:
The same problem as with `segments.join(".")` -- the segment might contain characters that are interpreted, such as a single quote or a backslash.

Reply:
> s/at //

Fixed.